src/Makefile.am

```diff
@@ -547,6 +547,7 @@ CLEANFILES += wallet/*.gcda wallet/*.gcno
 CLEANFILES += wallet/test/*.gcda wallet/test/*.gcno
 CLEANFILES += zmq/*.gcda zmq/*.gcno
 CLEANFILES += obj/build.h
+CLEANFILES += test/*.cpp.log wallet/test/*.cpp.log
```

Comment:
To avoid adding another nasty wildcard, I think this one can be $(BITCOIN_TESTS:.cpp=.cpp.log) .

Also, please append it to CLEAN_BITCOIN_TEST in Makefile.test.include instead, to keep the test stuff in one place.